nipype/interfaces/dcm2nii.py

```diff
@@ -292,7 +292,7 @@ def _parse_stdout(self, stdout):
                         bvals.append(out_file + ".bval")
                         find_b = False
                 # next scan will have bvals/bvecs
-                elif 'DTI gradient directions' in line:
+                elif 'DTI gradients' in line:
```

Comment:
Oh,
that's nasty. I thought 'DTI gradient directions' was the old dcm2nii
syntax and copy&paste introduced this error, but now I realize that this
is "old" dcm2niiX output.

+1 for another elif statement


> In nipype/interfaces/dcm2nii.py
> <https://github.com/nipy/nipype/pull/1463#discussion_r61746680>:
> 
>> @@ -292,7 +292,7 @@ def _parse_stdout(self, stdout):
>>                          bvals.append(out_file + ".bval")
>>                          find_b = False
>>                  # next scan will have bvals/bvecs
>> -                elif 'DTI gradient directions' in line:
>> +                elif 'DTI gradients' in line:
> 
> would suggest changing to |elif 'DTI gradient' in line:| to make
> compatible with former output as well